crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -1489,9 +1489,8 @@ where
             let root_blocks = self.root_blocks.lock();
             // If there are root blocks expected to be included in this block, check them
             if let Some(correct_root_blocks) = root_blocks.peek(&block_number) {
-                // TODO: Why there could be no body? Light client?
                 let mut found = false;
-                for extrinsic in block.body.as_ref().unwrap() {
+                for extrinsic in block.body.as_ref().expect("Block body must exist; qed") {
```

Comment:
Also this "qed" is not correct, "Block body must exist" itself doesn't prove why block body always exists exactly: https://en.wikipedia.org/wiki/Q.E.D.

Reply:
Updated. Feel free to suggest a better proof :(